wled00/fcn_declare.h

```diff
@@ -78,7 +78,8 @@ class NeoGammaWLEDMethod {
 };
 #define gamma32(c) NeoGammaWLEDMethod::Correct32(c)
 #define gamma8(c)  NeoGammaWLEDMethod::rawGamma8(c)
-[[gnu::hot]] uint32_t color_blend(uint32_t,uint32_t,uint16_t,bool b16=false);
+[[gnu::hot]] uint32_t color_blend(uint32_t,uint32_t,uint16_t);
+inline uint32_t color_blend(uint32_t c1, uint32_t c2, uint8_t blend)  { return color_blend(c1, c2, uint16_t(((uint16_t)blend << 8))); };
```

Comment:
The difference between uint8_t and uint16_t is very subtle - when writing new code it might be too "easy" to accidentally call the wrong function and get unexpected results.

As we already need to correct almost all occurrences - how about naming the 8bit version "color_blend" (legacy support), while using "color_blend16" for the 16bit one? It might prevent some mistakes, because we don't need to be careful about casting the last parameter to uint8_t or uint16_t.

Reply:
that is a very good suggestion, this was something I did not like about the change.